content/en/docs/tasks/administer-cluster/kubeadm/adding-windows-nodes.md

```diff
@@ -0,0 +1,160 @@
+---
+title: Adding Windows worker nodes
+content_type: task
+weight: 50
+---
+
+<!-- overview -->
+
+This page explains how to add Windows worker nodes to a kubeadm cluster.
+
+## {{% heading "prerequisites" %}}
+
+* A running [Windows Server 2019](https://www.microsoft.com/en-us/cloud-platform/windows-server-pricing)
+(or higher) instance with administrative access.
+* If you are using VXLAN/Overlay networking you must have also have
+[KB4489899](https://support.microsoft.com/help/4489899) installed.
+* A running kubeadm cluster created by `kubeadm init` and following the steps
+in the document [Creating a cluster with kubeadm](/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm/).
+* `curl.exe` is installed and present in the `PATH` environment variable.
```

Comment:
I think we can drop this point too. Curl.exe has been included in Windows since Windows Server 2019 which is the minimum target we have for Windows nodes.